about.html

```diff
@@ -0,0 +1,130 @@
+        <div class="mobile-menu-header">
+          <button type="button" class="close-button">&times;</button>
+        </div>
+        <nav>
+          <a href="./index.html" class="navigate">
+            Home
+          </a>
+          <a href="./about.html" class="navigate">
+            About
+          </a>
+          <a href="./index.html#program" class="navigate">
+            Program
+          </a>
+          <a href="#" class="navigate">
+            Join
+          </a>
+          <a href="./about.html#sponsor" class="navigate">
+            Sponsor
+          </a>
+        </nav>
+      </div>
+      <header>
+        <div class="social-media">
+          <nav class="social-nav">
+            <a href="https://facebook.com/edith.naisanga"><img src="./images/facebook-icon.png" alt="facebook"></a>
+            <a href="https://twitter.com/edithnaisanga"><img src="./images/twitter-icon.png" alt="twitter"></a>
+            <a href="#">English</a>
+            <a href="https://github.com/nedith">My page</a>
+            <a href="#">Logout</a>
+          </nav>
+        </div>
+        <div class="navbar">
+          <div class="navbar-logo-div"><a href="./index.html"><img class="summit-logo" src="./images/logo.jpg" alt=""></a></div>
+    
+          <nav class="navigation">
+            <a href="./about.html">About</a>
+            <a href="./index.html#program">Program</a>
+            <a href="">Join</a>
+            <a href="#sponsor">Sponsor</a>
+            <button class="ug-btn" type="button">UG Campaign</button>
+          </nav>
+    
+        </div>
+    
+      </header>
+
+      <main class="main">
+        <div class="about-box">
+          <button type="button" class="hamburger">
+            <img src="./images/menu.png" alt="menu icon">
+          </button>
+          <p class="hello">"Hello! Tourism World"</p>
+          <h1 class="heading">VISIT UGANDA <br> GLOBAL SUMMIT 2022</h1>
+          <section class="about-intro">
+            <h2 class="mission">The global summit brings together tourist experts, acamedics and enthusiasts
+                 who comprise of a wide network across the globe</h2>
+                 <p class="contact">Please contact us per Email for further questions<br> 
+                    about Visit Uganda Summit 2022 <br><span class="email">nedith22@gmail.com</span>
+                  </p>
+                  <h3 class="logo-heading">Visit Uganda Logo</h3>
+                  <p class="logo-creation">The visit Uganda logo was designed by the school of graphics design, Kampala</p>
+                  <div class="logo-container">
+                    <div class="logo"><img class="img-logo" src="./images/logo.jpg" alt="Visit Uganda logo"></div>
+                    <div class="logo-text">
+                      <p class="summit">VISIT UGANDA <br> SUMMIT 2022</p>
+                      <p class="period">3rd - 7th Oct 2022 | Finland</p>
+                    </div>
+                  </div>
```

Comment:
According to the project requirements, this should be a separate `<section>`. Please fix it.

<img width="908" alt="Screen Shot 2022-06-22 at 9 13 52" src="https://user-images.githubusercontent.com/82426682/175051303-bbfcbeca-7db7-4e89-a402-15143e2f4748.png">